platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/SettingConstants.java

```diff
@@ -72,6 +72,7 @@ public final class SettingConstants {
     static final boolean LOAD_KEYS_FROM_PFX_FILES_DEFAULT_VALUE = true;
     static final int MAX_TRANSACTION_BYTES_PER_EVENT_DEFAULT_VALUE = 245760;
     static final int MAX_TRANSACTION_COUNT_PER_EVENT_DEFAULT_VALUE = 245760;
+    static final boolean TRANS_THROTTLE_DEFAULT_VALUE = false;
```

Comment:
`transThrottle` was completely removed from the platform, so this constant should not be added.